Source/Core/DolphinQt/Debugger/MemoryWidget.cpp

```diff
@@ -689,6 +693,53 @@ void MemoryWidget::OnSetValue()
   Update();
 }
 
+void MemoryWidget::OnSetValueFromFile()
+{
+  if (!Core::IsRunning())
+    return;
+
+  auto target_addr = GetTargetAddress();
+
+  if (!target_addr.is_good_address)
+  {
+    ModalMessageBox::critical(this, tr("Error"), tr("Bad address provided."));
+    return;
+  }
+
+  if (!target_addr.is_good_offset)
+  {
+    ModalMessageBox::critical(this, tr("Error"), tr("Bad offset provided."));
+    return;
+  }
+
+  QString path = QFileDialog::getOpenFileName(this, tr("Select a file"), QDir::currentPath(),
+                                              QStringLiteral("All files (*.*)"));
```

Comment:
```suggestion
                                              tr("All files (*.*)"));
```

So that this can be translated for other languages

Reply:
`(*.*)` ignores files that don't have an extension, it would be better to just have a `(*)`.

I'm also a fan of leaving that part untranslated (eg. `QStringLiteral("%1 (*)").arg(tr("All Files"))` because we've had issues in the past where some translation file would break the file pattern, but ymmv.